app/javascript/controllers/dropzone_controller.js

```diff
@@ -31,6 +31,12 @@ export default class extends Controller {
     }
   }
 
+  isVisible (elem) {
+    // cobbled together from a combo of Jeremy's solution in https://github.com/sul-dlss/happy-heron/pull/1582 (commit 27c81bc2a) and
+    // this stack overflow discussion, particularly this thread: https://stackoverflow.com/a/33456469
+    return !!(elem.offsetParent || elem.offsetWidth || elem.offsetHeight || elem.getClientRects().length) && window.getComputedStyle(elem).visibility !== 'hidden'
```

Comment:
@jmartin-sul I wonder if there's enough [browser support](https://caniuse.com/mdn-api_element_checkvisibility) now to consider the solution outlined in https://stackoverflow.com/a/72717388/24039837?

Reply:
oh, possibly, thanks! i'll give this a shot in a bit, and post a follow up PR if it works out.

Reply:
@jmartin-sul awesome, thx!

Reply:
fixed in https://github.com/sul-dlss/happy-heron/pull/3604